src/test/goldens/polymer/lib/legacy/legacy-element-mixin.d.ts

```diff
@@ -238,7 +238,7 @@ declare namespace Polymer {
      *  `node` on which to fire the event (HTMLElement, defaults to `this`).
      * @returns The new event that was fired.
      */
-    fire(type: string, detail?: any, options?: any): Event|null;
+    fire(type: string, detail?: any, options?: {bubbles: boolean|undefined, cancelable: boolean|undefined, composed: boolean|undefined}): Event|null;
```

Comment:
Nit: |undefined can be an optional parameter

Reply:
Done.